README.md

```diff
@@ -227,6 +227,42 @@ jobs:
           sarif_file: 'trivy-results.sarif'
 ```
 
+### Using Trivy to generate SBOM
+It's possible for Trivy to generate an SBOM of your dependencies and submit them to a consumer like GitHub Dependency Snapshot.
+
+The sending of SBOM to GitHub feature is only available if you currently have [GitHub Dependency Snapshot](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) available to you in your repo. 
+
+In addition to send results to the GitHub Dependency Snapshot, you will need to create a [GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
```

Comment:
```suggestion
In order to send results to the GitHub Dependency Snapshot, you will need to create a [GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
```

Reply:
@knqyf263 - `GITHUB_TOKEN` doesn't have the `repo` scope as @achton mentioned. Here are the docs https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Here's a failed run to demo it: https://github.com/simar7/package-test/runs/6972488098?check_suite_focus=true

Reply:
@knqyf263 - thanks for the idea, I like it. I've implemented like you mentioned, here's an example:

```
      - name: Run Trivy in GitHub SBOM mode and upload results
        uses: aquasecurity/trivy-action@master
        with:
          scan-type: 'sbom'
          format: 'github'
          output: 'dependency-results.sbom.json'
          artifact-type: 'fs'
          image-ref: '.'
          github-pat: ${{ secrets.PAT_TOKEN }}
```